packages/coordinator/src/screens/AppoinmentScreenV2/AppointmentScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState } from "react";
+import styles from "./AppointmentScreen.module.scss";
+
+import Button, { ButtonVariant } from "../../components/Button";
+import { useNavigate } from "react-router-dom";
+import DatePicker from "../../components/V2/DatePicker";
+import TimePicker from "../../components/V2/TimePicker";
+import Select from "../../components/V2/Select";
+import { SelectOption } from "@zm-blood-components/common";
+import classnames from "classnames";
+
+export interface AppointmentScreenProps {
+  onSubmit: (date: Date, hour: Date, numberOfPlaces: number) => void;
+}
+
+const slotOptions: SelectOption<number>[] = [
+  1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
+].map((n) => ({
+  label: n + "",
+  value: n,
+  key: n + "",
+}));
+
+export default function AppointmentScreen(props: AppointmentScreenProps) {
```

Comment:
The screen name should be AddAppointmentScreen, no?

Reply:
done